e2e/support/helpers/e2e-downloads-helpers.ts

```diff
@@ -89,10 +93,15 @@ export function downloadAndAssert(
   cy.log(`Downloading ${fileType} file`);
 
   if (isDashboard) {
-    getDashboardCardMenu().click();
-    cy.findByText("Download results").click();
+    if (isEmbed) {
```

Comment:
We only show the `download-dashcard-results-button` in the embed UI, we still continue to show the dashcard menu in the main app. The reason is that there is more things you can do in that dashcard menu when in the main app, such as editing the questions.